File: HLCObjKotlinAct4/src/Java/Act4.java
package Java;

import java.util.Random;
import java.util.Scanner;

public class Act4 {
    public static void main(String[] args) {
        Random random = new Random();
        Scanner sc = new Scanner(System.in);
        int numRandom = random.nextInt(100) + 1;
        int tried;
        boolean guessed = false;
        do{
            System.out.println("Adivina el numero");
            tried = sc.nextInt();
            sc.nextLine();
            if(tried == numRandom){
                guessed = true;
            }else if(tried < numRandom){
                System.out.println("El numero es más grande");
            }else{
                System.out.println("El numero es más pequeño");
            }
        }while(!guessed);
        System.out.println("Has adivinado el numero");
    }
}

File: HLCObjKotlinAct4/src/Kotlin/Act4.kt
package Kotlin

import java.util.Random

fun main() {
    val random = Random()
    val numRandom = random.nextInt(100) + 1
    var tried: Int
    var guessed = false
    do {
        println("Adivina el numero")
        tried = readln().toInt()
        if (tried == numRandom) {
            guessed = true
        } else if (tried < numRandom) {
            println("El numero es más grande")
        } else {
            println("El numero es más pequeño")
        }
    } while (!guessed)
    println("Has adivinado el numero")
}

